srcpkgs/jetbrains-jdk-bin/template

```diff
@@ -10,8 +10,9 @@ license="GPL-2.0-only, Classpath-exception-2.0"
 homepage="https://github.com/JetBrains/JetBrainsRuntime"
 _jdk_ver=${version%b*}
 _jdk_build=${version#*b}
-distfiles="https://cache-redirector.jetbrains.com/intellij-jbr/jbr_jcef-${_jdk_ver//\./_}-linux-x64-b${_jdk_build}.tar.gz"
-checksum=6dc1208c486ed023ddd9f79f66f099304cd6785895d4e782aaefe0603f47e621
+_jdk_rev=_1
```

Comment:
Anything wrong with `version=11.0.14.1b2043.14` instead?

Reply:
xbps is unhappy with that format.
Am not on a machine where I can test this right now, but I could provide the actual error in a few days, if needed.

Reply:
Well, it _wasnt_ working for me, but I couldnt repro it now.
Changed back to the usual string.